cross-modal-search/flows/flow-index.yml

```diff
@@ -34,9 +34,25 @@ pods:
     needs: [gateway]
   - name: text_indexer
     polling: any
-    uses: pods/index-text.yml
+    #uses: pods/index-text.yml (numpy + binary pb indexer)
```

Comment:
Could you remove commented lines that are not necessary?

Reply:
Sure